pxtpy/pydecompiler.ts

```diff
@@ -40,6 +40,11 @@ namespace pxt.py {
         throw err;
     }
 
+    ///
+    /// FLAGS
+    ///
+    const SUPPORT_LAMBDAS = false;
```

Comment:
I'm guessing his is non-permanent / just to make it easier to find in the future when we do support lambdas, right?

Reply:
Yes exactly.